### PR TITLE
fix(ui): updated icon to be consistent with glossary

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes:

updated icon of version, follow and manage buttons as per the new style

image

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx [hunk 1]
```diff
@@ -385,53 +359,50 @@ const EntityPageInfo = ({
...
-                    {getVersionButton(toString(version))}
-                  </Tooltip>
-                ) : (
-                  <>{getVersionButton(toString(version))}</>
-                )}
-              </>
-            ) : null}
+            {!isUndefined(version) && (
+              <VersionButton
+                className="m-l-xs tw-px-1.5"
```
Comment (aniketkatkar97) [inline openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx:364]: Can we avoid the tailwind styling here?

File: openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx [hunk 2]
```diff
@@ -179,7 +185,7 @@ const EntityPageInfo = ({
     [tags]
   );
 
-  const getFollowers = () => {
+  const getFollowers = useMemo(() => {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx:188]:
```suggestion
  const followers = useMemo(() => {
```

**Review by harshach — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx [hunk 3]
```diff
@@ -385,53 +359,50 @@ const EntityPageInfo = ({
...
+                }
+                menu={{
+                  items: [
+                    {
+                      key: 'followers',
+                      label: getFollowers,
+                    },
+                  ],
+                }}
+                trigger={['click']}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx:389]: Not sure why we need dropdown button here, are we showing something on click here?

Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx:389]: Yes list of followers on click of count.

**Review by aniketkatkar97 — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**